addons/account/models/partner.py

```diff
@@ -538,6 +538,11 @@ def _get_company_currency(self):
     duplicated_bank_account_partners_count = fields.Integer(
         compute='_compute_duplicated_bank_account_partners_count',
     )
+    is_accounting_fields_required = fields.Boolean(compute="_compute_is_accounting_fields_required")
+
+    @api.depends('company_id')
```

Comment:
```suggestion
    @api.depends_context('company')
```
We don't really need a dependency here since changing the main company will trigger a reload and a recompute, but the `depends_context()` is more appropriate since `company_id` is not required.

Reply:
already tried it but the compute is not triggered at the opening of a new form,

Reply:
> depends_context() is more appropriate 

Already tried it but the compute is not triggered at the opening of a new form with that